js/app.js

```diff
@@ -0,0 +1,263 @@
+'use strict'
+
+var canvas = document.getElementById("myCanvas");
+var ctx = canvas.getContext("2d");
+var CANVAS_HEIGHT = canvas.height;
+var CANVAS_WIDTH = canvas.width;
+var TRACK_HEIGHT = CANVAS_HEIGHT/2;
+var TRACK_WIDTH = CANVAS_WIDTH;
+var TRACK_Y = CANVAS_WIDTH/8;
+var TRACK_X = 0;
+var CAR_WIDTH = 100;
+var CAR_HEIGHT = 50;
+var carOne = {
+  x: TRACK_X + 10,
+  y: TRACK_Y + 40
+};
+var carTwo = {
+  x: TRACK_X + 10,
+  y: TRACK_Y + 220
+};
+var dx = 2;
+var dy = -2;
+var rightKeyPressed = false;
+var leftKeyPressed = false;
+var upKeyPressed = false;
+var downKeyPressed = false;
+var dKeyPressed = false;
+var aKeyPressed = false;
+var wKeyPressed = false;
+var sKeyPressed = false;
+var raceStarted = false;
+var COUNT = 3;
+
+document.addEventListener("keydown", keyDownHandler, false);
+document.addEventListener("keyup", keyUpHandler, false);
+
+function keyDownHandler(event) {
+  if(event.keyCode === 39) {
+    rightKeyPressed = true;        
+  }
+
+  if(event.keyCode === 37) {
+    leftKeyPressed  = true;        
+  }
+  
+  // Allow the car to turn only when moving fordwards or backwards

```

Comment:
Allowing the car to only turn while moving is a really nice touch! I don't think I would have thought of that.